cmd/context/create.go

```diff
@@ -295,6 +295,16 @@ func getLoggedUserContext(ctx context.Context, c *ContextCommand, ctxOptions *Co
 		return nil, err
 	}
 
+	kubetokenClient, err := okteto.NewKubeTokenClient(ctxOptions.Context, ctxOptions.Token, ctxOptions.Namespace)
```

Comment:
NIT: I think this has to be injected from `ContextCommand` otherwise we'll be calling a real environment on tests